include/micm/solver/solver.hpp

```diff
@@ -63,6 +63,13 @@ namespace micm
       return solver_.Solve(time_step, state);
     }
 
+    // Overloaded Solve function to change parameters
+    SolverResult Solve(double time_step, StatePolicy& state, RosenbrockSolverParameters params)
+    {
+      solver_.parameters_.h_start_ = params.h_start_;
+      return solver_.Solve(time_step, state);
+    }
```

Comment:
```suggestion
    SolverResult Solve(double time_step, StatePolicy& state, RosenbrockSolverParameters params)
    {
      solver_.parameters_ = params;
      return solver_.Solve(time_step, state);
    }
```

We actually need **all** of the paramters to be updated. That was the impetus for moving the tolerances off of the params. Also, this needs to work for any set of parameters, not just rosenbrock